source/views/settings/screens/licenses/item.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import React, {Component} from 'react'
+import {Image, StyleSheet} from 'react-native'
+import {Detail, Title, ListRow} from '@frogpond/lists'
+import {Column, Row} from '@frogpond/layout'
+import {type LicenseType} from './types'
+import {type NavigationScreenProp} from 'react-navigation'
+
+type Props = {
+	item: LicenseType,
+	navigation: NavigationScreenProp<*>,
+}
+
+export class LicenseItem extends Component<Props> {
+	render() {
+		const {
+			key,
+			image,
+			username,
+			licenses,
+			name,
+			version,
+			licenseText,
+		} = this.props.item
+
+		let displayUsername = ''
+		if (username && key.toLowerCase() !== username.toLowerCase()) {
+			displayUsername = `published by ${username}`
+		}
+
+		return (
+			<ListRow
+				arrowPosition="center"
+				onPress={() =>
+					this.props.navigation.navigate('LicenseDetailView', licenseText)
+				}
+			>
+				<Row alignItems="center">
+					{image.length ? (
+						<Image
+							accessibilityIgnoresInvertColors={true}
+							source={{uri: image}}
+							style={styles.image}
+						/>
```

Comment:
Unfortunately, github doesn't redirect on username's avatar's like it does for repos, so it's possible to render a blank image here.